examples/swift/Package.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version: 5.9
```

Comment:
Can we use 5.5 as we used in the root Package.swift? Looks like Ci is failing because it uses Swift 5.7.

Reply:
✅